### PR TITLE
[ICD] Update root device type to include icdm cluster

### LINKED ISSUE #26798: [ICD] Add ICD features to BaseDeviceType and to RootNodeDevice Type
### Feature description

Capability map in BaseDeviceType should contain SIT and  LIT bits

### Platform

all

### Platform Version(s)

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by mkardous-silabs**
#### Description
Spec added the ICDM cluster to root node device type. 
Adding it in the SDK.

fixes #26798

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 1]
```diff
@@ -84,6 +84,11 @@ limitations under the License.
             <include cluster="Ethernet Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
             <include cluster="WiFi Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
             <include cluster="Thread Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="ICD Management" clients="false" server="false" clientLocked="true" serverLocked="false">
+                <requireAttribute>IDLE_MODE_DURATION</requireAttribute>
+                <requireAttribute>ACTIVE_MODE_DURATION</requireAttribute>
+                <requireAttribute>ACTIVE_MODE_THRESHOLD</requireAttribute>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:90]: Why do we need those?  Are those not marked as required in the cluster spec?  I think `requireAttribute` is for device-level overrides of the cluster specs....

Comment (mkardous-silabs) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:90]: @bzbarsky-apple Based on the OnOff light device type, everything that is mandatory is indicated with the `required...`.

**Review by jmartinez-silabs — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml (hunk 1 above)
Comment (bzbarsky-apple) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:90]: I think the OnOff device type XML is wrong, fwiw.  But maybe I don't understand how ZAP processes these things....

Comment (bzbarsky-apple) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:90]: Basically: Ideally the XML would reflect the spec here, and the spec does not have these requirements in the _device type_.

Comment (mkardous-silabs) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml]: removed the attributes

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 2]
```diff
@@ -84,6 +84,7 @@ limitations under the License.
             <include cluster="Ethernet Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
             <include cluster="WiFi Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
             <include cluster="Thread Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="ICD Management" clients="false" server="false" clientLocked="true" serverLocked="false"></include>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:87]:
```suggestion
            <include cluster="ICD Management" client="false" server="false" clientLocked="true" serverLocked="false"></include>
```

**Review by bzbarsky-apple — APPROVED** [review]

**Pull request merged**